src/audio/kpb.c

```diff
@@ -60,6 +60,7 @@ struct comp_data {
 	size_t hb_buffer_size;
 	size_t host_buffer_size;
 	size_t host_period_size;
+	bool sync_draining_mode;
```

Comment:
Can we add some doxygen comments to the entire KPB code (structures and functions). We now have some degree of complexity which is not easy to follow. This can be done as an incremental change in a new PR (before any other new KPB PRs are merged).

Reply:
@lgirdwood, okay I will put it on my TODO list.